pkg/lib/context-menu.scss

```diff
@@ -0,0 +1,17 @@
+.contextMenu {
+  position: fixed;
+  /* xterm accessibility tree has z-index 100 and we need to be in front of it
+     * to be able to handle mouse events.
+     */
+  z-index: 101;
+
+  &Option .pf-c-menu__item-text {
+    display: inline-flex;
+    justify-content: space-between;
+    min-width: 10rem;
```

Comment:
We should use gap with a PF spacer to ensure there's space between elements. Relying on min-width means the text will run into each other when strings get long.